### PR TITLE
Support logging expert balancedness metrics

### PR DESCRIPTION
**Pull request by fzyzcjy**
## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

File: python/sglang/srt/managers/expert_distribution.py [hunk 1]
```diff
@@ -472,7 +474,80 @@ def dump(self, output_mode: _OutputMode):
...
+            utilization_rate_tensor = compute_utilization_rate(gpu_physical_count)
+            utilization_rate = torch.mean(utilization_rate_tensor).item()
+            self._history.append(utilization_rate)
+
+            gpu_physical_count_sum = gpu_physical_count.sum().item()
+
+            logger.info(
+                f"[Expert Balancedness] "
+                f"forward_pass_id={forward_pass_id} "
+                f"current_pass_value={utilization_rate:.03f} "
```
Comment (yizhang2077) [inline python/sglang/srt/managers/expert_distribution.py]: I think we could replace value to balancedness for read-friendly? and `last_{size}_value` to `last_{size}_average_balancedness` is better?

**Review by yizhang2077 — APPROVED** [review]
I think logger metrics name could be better, apart from that LGTM

File: python/sglang/srt/managers/expert_distribution.py (hunk 1 above)
Comment (fzyzcjy) [reply python/sglang/srt/managers/expert_distribution.py]: looks reasonable, will change
